pom.xml

```diff
@@ -526,7 +526,6 @@
         <version.org.jboss.common.jboss-common-beans>2.0.1.Final</version.org.jboss.common.jboss-common-beans>
         <legacy.version.org.jboss.ejb-client>4.0.50.Final</legacy.version.org.jboss.ejb-client>
         <version.org.jboss.ejb-client>5.0.2.Final</version.org.jboss.ejb-client>
-        <version.org.jboss.ejb-client-legacy>3.0.3.Final</version.org.jboss.ejb-client-legacy>
```

Comment:
@tadamski Please also remove https://github.com/wildfly/wildfly/blob/main/pom.xml#L503 -- 'legacy.version.org.jboss.narayana'. The only use of that property was in testsuite/integration/legacy-ejb-client.pom.xml so this PR makes it cruft.

Reply:
fixed